test/integration/features/cmd-openshift.feature

```diff
@@ -48,7 +48,7 @@ cluster in VM provided by Minishift.
        And stdout should match "router-\d-\w{5}\s*1\/1\s*Running"
 
   Scenario: User deploys nodejs example application from OpenShift repository
-     Given Minishift has state "Running"
+     Given Openshift has state "Running"
```

Comment:
@jemacom This is Minishift only. :)

Reply:
This test scenario requires Openshift (that resides inside minishift ) to be running. Because you can have Minishift running and Openshift stopped and no oc command will work.

Reply:
Minishift needs to be in running state, and with this OpenShift should be running